content/en/docs/concepts/workloads/pods/sidecar-containers.md

```diff
@@ -0,0 +1,73 @@
+---
+title: API for sidecar containers
+content_type: concept
+weight: 50
+---
+
+<!-- overview -->
+{{< feature-state for_k8s_version="v1.28" state="alpha" >}}
+
+Sidecar containers are the secondary containers that runs along with the main
+application container within the same {{< glossary_tooltip text="Pod" term_id="pod" >}}.
+These containers are used to enhance or extend the functionality of the primary container
```

Comment:
Please **be consistent** when you are using a term. If you call something "main containers", or "application containers", then use it across the page. Do not introduce a term like "primary containers". This confuses the users.


```suggestion
These containers are used to enhance or to extend the functionality of the main application container
```

Reply:
Thanks for the suggestion, i will keep in mind.